GenderPayGap.WebUI/BackgroundJobs/HangfireConfiguration/HangfireConfigurationHelper.cs

```diff
@@ -13,7 +14,8 @@ public static class HangfireConfigurationHelper
 
         public static void ConfigureServices(IServiceCollection services)
         {
-            services.AddHangfire(x => x.UseSqlServerStorage(Global.DatabaseConnectionString));
+//            services.AddHangfire(x => x.UseSqlServerStorage(Global.DatabaseConnectionString));
+            services.AddHangfire(x => x.UsePostgreSqlStorage(@"Server=127.0.0.1;Port=5432;Database=gpg_database;User Id=gpg_user;Password=local_gpg_database;"));

```

Comment:
I think this should be based on the connection string?
Otherwise it might only work on our laptops (when the database is `127.0.0.1`)